app/controllers/mou_controller.rb

```diff
@@ -5,7 +5,12 @@ def index
   end
 
   def create
-    redirect_path = replacing? ? replaced_mou_index_path : created_mou_index_path
+    redirect_path = if replacing?
+                      replaced_mou_index_path(organisation: current_organisation)
+                    else
+                      created_mou_index_path(organisation: current_organisation)
+                    end
```

Comment:
@Renny4Real Looks like these changes are already in another PR: https://github.com/alphagov/govwifi-admin/pull/695/files

Did you branch off a feature branch rather than master?

Reply:
Yeah, this branch contains everything we need so closed the feature branch.